app/helpers/application_helper.rb

```diff
@@ -4,4 +4,10 @@ module ApplicationHelper
   def l(object, options = {})
     super(object, options) if object
   end
+
+  def navigation_item_class(path)
+    classes = ["govuk-header__navigation-item"]
+    classes << "govuk-header__navigation-item--active" if current_page?(path)
+    return classes.join(" ")
```

Comment:
If you remove this `return` standard will pass

Reply:
brilliant, thank you!